code/modules/surgery/stomachpump.dm

```diff
@@ -25,29 +25,29 @@
 //Working the stomach by hand in such a way that you induce vomiting.
 /datum/surgery_step/stomach_pump
 	name = "Pump Stomach"
-	accept_hand = TRUE
+	implements = list(/obj/item/reagent_containers/syringe = 100)
```

Comment:
can you add ghetto surgery equivalents for this step?

Reply:
Stab with pen and succ, 5% of reagents transfered in your stomach.

Reply:
I'd been struggling trying to think of a good ghetto equivalent- if you have a suggestion, I'd be happy to add it.

Reply:
Rapid piping device clearly.

Reply:
Plunger?

Reply:
you should be able to use your hand but the reagents get on your hand (you get touch reacts from the chems you're pulling out) ;)